linkis-computation-governance/linkis-manager/linkis-label-common/src/main/java/org/apache/linkis/manager/label/entity/engine/EngineType.scala

```diff
@@ -92,6 +92,9 @@ object EngineType extends Enumeration with Logging {
     case _ if APPCONN.toString.equals(str) => APPCONN
```

Comment:
I don't think the scalability is so good in this way. 
try to use Enumeration.withName replace it 

https://stackoverflow.com/questions/33593525/scala-safe-way-of-converting-string-to-enumeration-value